forms-shared/src/slovensko-sk/xmlBuilder.ts

```diff
@@ -1,15 +1,26 @@
 import { Builder } from 'xml2js'
+import { GenericObjectType } from '@rjsf/utils'
 
-const builderOptionsBase = {
-  xmldec: {
-    version: '1.0',
-    encoding: 'UTF-8',
+export const buildSlovenskoSkXml = (
+  xmlObject: GenericObjectType,
```

Comment:
would expect more then GenericObjectType - I assume it must contain at least this part:

```
 eform: {
      $: {
        xmlns: getSlovenskoSkXmlns(formDefinition),
        'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
      },
```

if not more. Could we capture that in types ?

Would also add jsdoc detailing where the output from this is supposed (expected) to be used

Reply:
Not really, the library accepts (in builder) / returns (in parser) `any`, I want to use it as universal builder, e.g. replace the one [here](https://github.com/bratislava/konto.bratislava.sk/blob/0ee249b929dce438eb83a6754d9d4d9347c5168a/nest-forms-backend/src/nases/utils-services/tokens.nases.service.ts#L465), which has vastly different output.